LICENSE.txt

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2014–2016 VisionApps, s.r.o.
+Copyright (c) 2014–2018 VisionApps, s.r.o.
```

Comment:
As of 1st January 2019 this will be wrong. I would remove the years if possible.

Reply:
In the US the copyright is not effective unless you put a year here (as far as I remember). It should stay here.